tools/make/tests.toml

```diff
@@ -31,3 +31,80 @@ description = "Run all Rust doctests with all features"
 category = "ICU4X Development"
 command = "cargo"
 args = ["test", "--all-features", "--doc", "--no-fail-fast"]
+
+[tasks.check-cargo-tutorial-local]
+description = "Build the Cargo tutorial projects based on local crates"
+category = "ICU4X Development"
+script_runner = "@duckscript"
+script = '''
+exit_on_error true
+
+pwd = pwd
+set_env CARGO_TARGET_DIR ${pwd}/target
+
+cd docs/tutorials/cargo_tests
+rm -f Cargo.lock
```

Comment:
I think for local we didn't want this? There's no point checking in the file and then always deleting it

Reply:
done